ex1/index.js

```diff
@@ -0,0 +1,7 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert degrees Celsius");
+const degreesCelsius = prompt(">");
+const calculate = (degreesCelsius * 9 / 5) + 32;
```

Comment:
Verbs are commonly used for functions that DO something. In this case, I'd suggest using something like `degreesInFahrenheit`. It better explains what is stored in that variable.